readme.md

```diff
@@ -28,11 +33,11 @@
 2. Normal run, for dev purpose
 
 ```
-docker compose -f docker-compose.dev.yml up --build
```

Comment:
docker-compose is an old command for docker cli. Perhaps you need to upgrade your docker version to use `docker compose` command instead of `docker-compose`

Pls refer to this page for more information https://docs.docker.com/compose/

Reply:
yes, you were right, I need to upgrade my docker compose with docker-compose-plugin, thanks for your information

sudo apt update
sudo apt-get install docker-compose-plugin